debian/control

```diff
@@ -28,6 +28,7 @@ Depends:
   python3-ftdi1,
   sosreport,
   sudo,
+  avahi-daemon,
```

Comment:
Avahi is not needed to run a RevPi. Is a hard dependency really what we want here? Would a `Recommends` or even a `Suggests` be better here. Not everyone wants avahi on his system.

Reply:
For the revpi-tools package I agree with @iluminat23 to put it as Suggests to the control file. But we have to TEST the package to find out what will happen if:
- No avahi-daemon installed
- Install revpi-tools (this will create the folder /etc/avahi/services)
- What will happen if I now install avahi-daemon and the location /etc/avahi already exist?

Reply:
Normaly this should not be a issue as the directoy will be created and the directory should be removed when the last package is removed that uses this directory